qemu/tests/cfg/blockdev_commit_cor.cfg

```diff
@@ -0,0 +1,23 @@
+- blockdev_commit_cor:
+    type = blockdev_commit_cor
+    virt_test_type = qemu
+    only Linux
+    start_vm = yes
+    kill_vm = yes
+    device_tag = "data"
+    images += " ${device_tag}"
+    force_create_image_data = yes
+    remove_image_data = yes
+    image_size_data = 500M
+    image_name_data = data
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    image_copy_on_read_data = True
```

Comment:
yes or on or true, which follows our current design

Reply:
Have updated it, thanks zhenchao,